frigg/builds/models.py

```diff
@@ -34,6 +34,7 @@ class Project(TimeStampModel):
     queue_name = models.CharField(max_length=200, default=settings.FRIGG_WORKER_QUEUE)
     can_deploy = models.BooleanField(default=False, db_index=True)
     should_clone_with_ssh = models.BooleanField(default=False)
+    image = models.CharField(max_length=200, default="")
```

Comment:
Should this have `blank=True`? I think that it may be necessary for using django admin.